### ISSUE TITLE
Restarting Neard doesn't work

### ISSUE THREAD

**Issue by lothereus** [crazy-max/neard#82]
While clicking Restart in Neard, it engage the restart procedure, Neard window opening saying it is restarting, but when Neard is off, the window disappear and it doesn't restart...

I'm on Windows 8 64bit.

This is the log and core/tmp file you ask : https://www.sendspace.com/file/802lmf

**Comment by crazy-max** [comment]
Starts neard, creates this file on your desktop called for example ``restart.bat`` and add this :

```
@ECHO OFF

PING 1.1.1.1 -n 1 -w 2000 > nul
"E:/Neard/neard.exe" -quit -id={neard}
"E:/Neard/core/libs/php/php-win.exe" "bootstrap.php" "restart"
```

Tell me if it works.

**Comment by lothereus** [comment]
The .bat file does the same thing than restart.

Ok I just understand what happen... the services restart but the Neard app quit and the tray icon disappear. I don't know if I say it well.

(au pire on se retrouve en Hangout si tu veux plus de détails)

**Comment by crazy-max** [comment]
I think RunFromProcess does not work on your system, can you try this command (with Neard shutdown) :

``E:/Neard/tools/runfromprocess/runfromprocess1.05/RunFromProcess-x64.exe nomsg explorer.exe E:\Neard\neard.exe``

**Comment by lothereus** [comment]
That does nothing...

**Comment by crazy-max** [comment]
Hum ok, you are on a 64 bits system yes ?

**Comment by crazy-max** [comment]
Can you try this command @lothereus :
``E:/Neard/tools/runfromprocess/runfromprocess1.05/RunFromProcess-x64.exe admin explorer.exe E:\Neard\neard.exe``

**Comment by crazy-max** [comment]
I figure out where this come from.
Will be fixed in the next release.

**Comment by crazy-max** [comment]
Solved on Neard 1.0.13
